app/controllers/events_controller.rb

```diff
@@ -0,0 +1,34 @@
+class EventsController < ApplicationController
+  def index
+    @events = Event.all
+  end
+
+  def new
+    @event = current_user.events.build
+  end
+
+  def create
+    @event = current_user.events.build(event_params)
+    if @event.save
+      flash[:notice] = 'Event Created Successfully'
+      redirect_to root_path
+    else
+      flash.now[:alert] = 'Form not submitting for some reasons'
+      render :new
+    end
+  end
```

Comment:
### Pass in event_params method as private and list all the attributes
```
  def event_params
    params.require(:event).permit(:attr1, :attr2)
  end
```

Reply:
They are all listed in the application_controller, we felt its keeps our controller clean and neat